Makefile

```diff
@@ -72,12 +72,15 @@ echo-bash:
 	@echo "TL/DR: add these lines to run ~/.bashrc"
 	@echo "export PULP_SDK_HOME=${PULP_SDK_INSTALL_DIR}"
 	@echo "export LLVM_INSTALL_DIR=${LLVM_INSTALL_DIR}"
-	@echo "export PULP_RISCV_GCC_TOOLCHAIN=/PULP_SDK_IS_A_MESS"
+	@echo "export PULP_RISCV_GCC_TOOLCHAIN="
```

Comment:
Why not completely remove this?

Reply:
The pulp-sdk needs the environment variable set afaik but @Scheremo could confirm.
I would prefer if the contents of the variable stay untouched :smirk:

Reply:
Yes but it gets set when you source the platform specific setup script in the PULP SDK. I am certain that `PULP_SDK_IS_A_MESS` won't fix any problems 😉

Reply:
This is some deep PULP lore material here 😁

Reply:
WTF, I guess I was wrong 😂 Fight fire with fire...
I thought this was just a rant, but it is actually a fix 😂

Reply:
@Xeratec you underestimated pulp *and* Moritz, tsk tsk tsk